src/components/ImageView/index.native.js

```diff
@@ -69,6 +69,13 @@ class ImageView extends PureComponent {
                 imageWidth = Math.round(this.props.windowWidth);
                 imageHeight = Math.round(imageWidth * aspectRatio);
             }
+            const maxZoomRatio = 11;
+            if (width > (this.props.windowWidth * maxZoomRatio)) {
+                imageWidth = (this.props.windowWidth * maxZoomRatio);
+            }
+            if (height > (this.props.windowHeight * maxZoomRatio)) {
+                imageHeight = (this.props.windowHeight * maxZoomRatio);
```

Comment:
There is one issue with these. it won't preserve the aspect ratio of the image.  As soon as we cross the limit it will take the max resolution, defined here.

Reply:
@parasharrajat  we already have resized mode `contain` so it will save the image aspect ratio
and the video shows that. 

https://github.com/Expensify/App/blob/096c97917a40ca820c1ce83b28743d05288d746a/src/components/ImageView/index.native.js#L119

also, I think I need to delete this part: -

```ts 
if (width > (this.props.windowWidth * maxZoomRatio)) {
    imageWidth = (this.props.windowWidth * maxZoomRatio);
}
```
since there is resize for width before it if the width more than screen width 
https://github.com/Expensify/App/blob/096c97917a40ca820c1ce83b28743d05288d746a/src/components/ImageView/index.native.js#L68

Reply:
> we already have resized mode contained so it will save the image aspect ratio
and the video shows that.

Ok. Nice. Better leave a comment here about it. 

> since there is resize for width before it if the width more than screen width 

Oh, It reminds me that we possibly have another issue that does something like this. I can't think of which one but lets proceed with your solution and we can dry that later on that issue.